.travis.yml

```diff
@@ -5,9 +5,10 @@ language: ruby
 sudo: false
 
 rvm:
-  - &ruby1 2.3.1
-  - &ruby2 2.2.5
-  - &ruby3 2.1.9
+  - &ruby1 2.4.0
+  - &ruby2 2.3.1
+  - &ruby3 2.2.5
+  - &ruby4 2.1.9
```

Comment:
that's for a separate PR. This PR is only about Ruby 2.4.  
You're welcome to create a PR to bump the rvm versions we currently test against.  
:smiley: